<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportImageDialog</class>
 <widget class="QWidget" name="ExportImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>170</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="legendGroupBox">
     <property name="title">
      <string>Legend Configuration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QCheckBox" name="addLegendCheckBox">
        <property name="text">
         <string>Add legend to exported sky image</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="horizontalSpacing">
         <number>3</number>
        </property>
        <property name="verticalSpacing">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="legendOrientationLabel">
          <property name="text">
           <string>Legend orientation:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="legendOrientationComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="legendTypeLabel">
          <property name="text">
           <string>Legend type:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="legendTypeComboBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="legendPositionLabel">
          <property name="text">
           <string>Legend position:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="legendPositionComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
